pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1327,4 +1327,9 @@ self: super: {
   haskell-lsp_0_18_0_0 = super.haskell-lsp_0_18_0_0.override { haskell-lsp-types = self.haskell-lsp-types_0_18_0_0; };
   lsp-test_0_8_2_0 = (dontCheck super.lsp-test_0_8_2_0).override { haskell-lsp = self.haskell-lsp_0_18_0_0; };
 
+  # https://github.com/jyp/glpk-hs/pull/11
+  # patch is the above PR with conflicts resolved
+  glpk-hs = assert super.glpk-hs.version == "0.7"; 
+    appendPatch super.glpk-hs ./patches/glpk-hs-ghc86.patch;
```

Comment:
If possible, in nixpkgs we generally prefer to pull patches from upstream directly with `fetchpatch`.  There should be some examples of doing this in this file.

Can you rewrite this to use fetchpatch?

Reply:
That was exactly my previous approach, but the PR in its current form has merge conflicts with 0.7 and the patching fails. If there is a cleaner solution other than creating a patch manually, I would certainly use it.

Reply:
@pschuprikov Could you send the patch in this PR as a PR to upstream?